### PR TITLE
[ktor-server] fix: encoding with default content-type

### PR DESCRIPTION
**Pull request by rocketraman**
### :pencil: Description

For the ktor server, fixes the default encoding used to read the request body when the client does not specify an encoding in the content type explicitly.

### :link: Related Issues

https://github.com/ExpediaGroup/graphql-kotlin/issues/1672

### CODE REVIEW

File: servers/graphql-kotlin-ktor-server/src/main/kotlin/com/expediagroup/graphql/server/ktor/KtorGraphQLRequestParser.kt [hunk 1]
```diff
@@ -23,8 +23,9 @@ import com.fasterxml.jackson.databind.ObjectMapper
 import com.fasterxml.jackson.databind.type.MapType
 import com.fasterxml.jackson.databind.type.TypeFactory
 import io.ktor.http.HttpMethod
-import io.ktor.server.request.ApplicationRequest
-import io.ktor.server.request.receiveText
+import io.ktor.server.request.*
```
Comment (dariuszkuc) [inline servers/graphql-kotlin-ktor-server/src/main/kotlin/com/expediagroup/graphql/server/ktor/KtorGraphQLRequestParser.kt:26]: ktlint complains about the wildcard imports -> can you fix it?

Comment (rocketraman) [reply servers/graphql-kotlin-ktor-server/src/main/kotlin/com/expediagroup/graphql/server/ktor/KtorGraphQLRequestParser.kt]: Yep, done in https://github.com/ExpediaGroup/graphql-kotlin/pull/1673/commits/a5f75e17d4a270b947aa52ddbee692ad5f66c05c

**Comment by rocketraman** [comment]
@dariuszkuc Ok, a couple of tests failed. While investigating their resolution, I noticed that ktor 2.x now apparently no longer has the same behavior -- `receiveText` does now default to UTF-8 as of:

https://github.com/ktorio/ktor/commit/5121d10292a3dbe256fcc050fb155b46c965ae91
https://youtrack.jetbrains.com/issue/KTOR-4426

Therefore I would say this PR is no longer required.

**Comment by dariuszkuc** [comment]
Thanks for the update!

**Pull request closed without merging**